.github/workflows/ci.yaml

```diff
@@ -0,0 +1,38 @@
+name: ci
+on:
+  release:
+    types: [published]
```

Comment:
This action will build and push an image named `buger/goreplay:<tagName>` on Dockerhub every time a GitHub release is published.